hs_labels/migrations/0003_auto_20160107_1847.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
Consider changing the migration file name (e.g, 0003_custom_migration_20160107_1847.py) as the current name implies it was automatically generated by Django.